index.html

```diff
@@ -309,6 +309,25 @@ <h2>Skills</h2>
 					</ul>
 				</div>
 			</section>
+			
+			<section class="contact_section">
+				<div class="contact_section_container">
+					<h2>Contact me</h2>		
+					<div class="contact_text">
+						<p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+					</div>
+					<form action="https://formspree.io/f/mbjbpwzk" method="POST" autocomplete="on">						
+							<input type="text" id="name" name="name" placeholder="Name" maxlength="30"><br>						
```

Comment:
- [ ] As requested by the previous reviewer you should add the `required` attribute to each of your input fields for validation reasons so that a user can not submit empty data. But as you can see you haven't added the `required` attribute to this input field.

Reply:
Well noted.